### ISSUE TITLE
Get App on Fdroid

### ISSUE THREAD

**Issue by codepony** [vanita5/twittnuker#2]
Would be nice to have twittnuker in an alternative app-store to get automatic upgrades.

**Comment by vanita5** [comment]
F-Droid Repo: http://twittnuker.org/repo/

**Comment by rodneyrod** [comment]
Tried that link but it apparently can't find the index file, and I think F-Droid is deprecating any repos that aren't on HTTPS or those which don't have fingerprints.

**Comment by vanita5** [comment]
You can try https://twittnuker.org/repo/ (HTTPS).
For me it's working even without HTTPS

**Comment by 0x1f415** [comment]
looks like your certificate is invalid

![4-30-2016 2 21 06 pm](https://cloud.githubusercontent.com/assets/6064300/14937734/e3761e1c-0ede-11e6-8f40-c6f14e62b91e.png)

**Comment by vanita5** [comment]
The certificate has been fixed. Thanks for pointing this out!

**Comment by vanita5** [comment]
Note: I discontinued work on the f-droid branch some time ago. It's too much work for me atm.
If anyone is interested in maintaining the f-droid branch, feel free to contact me!

**Comment by sternenseemann** [comment]
What is the problem getting it into the main repository? Twidere is in there, too.

**Comment by vanita5** [comment]
The main problem is maintining the f-droid branch. Non-FOSS dependencies and code has to be removed/replaced.

**Comment by sternenseemann** [comment]
Wouldn't it make sense to do this even in the main distribution?

**Comment by vanita5** [comment]
yeah, maybe. Currently, it relies on GCM/Firebase and the Play Services for push notifications, donations and some other stuff.

**Comment by IzzySoft** [comment]
@vanita5 would indeed be good to get rid of that, generally 😄 As for now your app is [in my repo](https://apt.izzysoft.de/fdroid/index/apk/de.vanita5.twittnuker), but your development seems to be stalled: there were a bunch of commits after the last release. Mind to at least make that to another release? And are you still maintaining this app?

**Comment by eliah-w** [comment]
@IzzySoft I won't be working on this project again anytime soon, sorry.

**Comment by IzzySoft** [comment]
Not even making a "final release" until you find time to pick up again?
